web/app/views.py

```diff
@@ -224,6 +207,15 @@ def train():
         else:
             args.gpu = int(gpu)
 
+    current_user = request.cookies.get('currentUser')
+
+    if not current_user:
+        # Use DEFAULT as current user if the client's cookie is not set.
+        current_user = 1
```

Comment:
Can you make this a global constant, similar to my suggestion for the default user id?